src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -609,7 +612,19 @@ private void OnDiscoveryMessageReceived(ITestDiscoveryEventsHandler2 discoveryEv
             OnDiscoveryAbort(discoveryEventsHandler, ex, false);
         }
     }
-
+    private static bool BlameIsEnabled()
+    {
+        string? s = RunSettingsManager.Instance.ActiveRunSettings.SettingsXml;
+        DataCollectionRunSettings dataCollectionRunSettings = XmlRunSettingsUtilities.GetDataCollectionRunSettings(s)!;
```

Comment:
Looks like the method can return `null`, I would recommend to handle it rather than "expecting" result to not be `null`.